README.md

```diff
@@ -183,10 +183,32 @@ With reference to the [GitHub Commit status API], the `POST` parameters (`state`
 - `repo`\
   The GitHub repository name.
 
+
+###  Authentication keys
+
+Keys must be provided for one of the following authentication methods:
+1. [Personal Access Token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens)
+1. [GitHub app](https://docs.github.com/en/apps/creating-github-apps/about-creating-github-apps/about-creating-github-apps)
+
+#### Personal Access Token keys
+
 - `access_token`\
   The OAuth access token.\
   See also: section [GitHub OAuth token](#github-oauth-token).
 
+#### GitHub app keys
+
+- `private_key` \
+  The private key generated for the GitHub application
+
+- `application_id`
```

Comment:
According to the GitHub docs... this can be app id or client id... client id is preffered.

Reply:
client id is not yet supported by https://github.com/bradleyfalzon/ghinstallation/issues/118